src/diffCheckBindings.cc

```diff
@@ -74,6 +75,10 @@ PYBIND11_MODULE(diffcheck_bindings, m) {
         .def(py::init<>())
         .def(py::init<std::vector<Eigen::Vector3d>, std::vector<Eigen::Vector3i>, std::vector<Eigen::Vector3d>, std::vector<Eigen::Vector3d>, std::vector<Eigen::Vector3d>>())
 
+        .def("compute_distance", &diffCheck::geometry::DFMesh::ComputeDistance,
```

Comment:
same question if we still need it since you use RhinoCommons

Reply:
Can be deleted but I let you do that not to break anything on the c++ side..